main/modes/mode_dance.c

```diff
@@ -1299,44 +1299,26 @@ void danceRainbowSolid(uint32_t tElapsedUs, uint32_t arg __attribute__((unused))
 }
 
 /**
- * Turn on all LEDs and Make Purely White
+ * Turn on all LEDs to pure white
  *
  * @param tElapsedUs The time elapsed since last call, in microseconds
  * @param reset      true to reset this dance's variables
  */
 void danceFlashlight(uint32_t tElapsedUs, uint32_t arg __attribute__((unused)), bool reset)
 {
-    static uint32_t tAccumulated = 0;
-
-    if(reset)
-    {
-        tAccumulated = 70000;
-        return;
-    }
-
     // Declare some LEDs, all off
     led_t leds[NUM_LEDS] = {{0}};
-    bool ledsUpdated = false;
 
-    tAccumulated += tElapsedUs;
-    while(tAccumulated >= 70000)
```

Comment:
I believe this was intentional so that brightness changes would get picked up. Please test, verify, and if that's true, undo

Reply:
Yeah, I tested out doing it this way a few days back and it didn't update when the brightness changed.

Reply:
I tested before making the PR, and brightness changes are working properly. I must have stripped more out than you did @dylwhich. Re-cleaning, flashing, and testing again just to be absolutely certain

Reply:
Can confirm, working properly with brightness on this branch!

Reply:
Double confirmed, works fine here. But thinking about it, it works because the LEDs are reset every loop iteration, when they had been updated every 70ms. Code cleanliness came at the cost of a busier CPU and peripherals, which is not good for something that's relatively static.. I'm not going to merge this branch, but I will disable the flashlight in main